README.md

```diff
@@ -38,14 +38,14 @@ Eventually Dreamboat will vanish, as if a dream, once in-protocol PBS is live.
 
 ## Design Goals
 
-Special thanks to Flashbots for their commiment to open source. This project was orginally based on the relay in Flashobot's [boost-geth-builder](https://github.com/flashbots/boost-geth-builder) and [mev-boost-relay](https://github.com/flashbots/mev-boost-relay).  We hope to continue to diverge in implementation over time to help strengthen #RelayDiversity as we continue to work on Dreamboat's core design goals:
+This is a from-scratch implementation of a PBS relay – aimed at strengthening #RelayDiversity – with the following design goals:
```

Comment:
This is a from-scratch implementation of a PBS relay—aimed at strengthening #RelayDiversity—with the following design goals: